### ISSUE TITLE
[GitHub] [Med] Regular expressions Cross-Site Scripting (XSS) vulnerability (due 4/7/20)

### ISSUE THREAD

**Issue by lbeaufort** [fecgov/openFEC#4198]
Remediation
Upgrade serialize-javascript to version 2.1.1 or later. For example:
```
"dependencies": {
  "serialize-javascript": ">=2.1.1"
}
```
or…
```
"devDependencies": {
  "serialize-javascript": ">=2.1.1"
}
```
Always verify the validity and compatibility of suggestions with your codebase.

Details
GHSA-h9rv-jmmf-4pgx
moderate severity
Vulnerable versions: < 2.1.1
Patched version: 2.1.1
regular expressions Cross-Site Scripting (XSS) vulnerability
Impact
Affected versions of this package are vulnerable to Cross-site Scripting (XSS). It does not properly mitigate against unsafe characters in serialized regular expressions.

This vulnerability is not affected on Node.js environment since Node.js's implementation of RegExp.prototype.toString() backslash-escapes all forward slashes in regular expressions.

If serialized data of regular expression objects are used in an environment other than Node.js, it is affected by this vulnerability.

Patches
This was patched in v2.1.1.

**Comment by pkfec** [comment]
@PaulClark2 given the due date (4/7/2020) we can bump this issue to Innovation sprint.

**Comment by jason-upchurch** [comment]
This was not flagged by snyk because the vulnerability is in package-lock.json devDependencies.

The vulnerability is verified locally by `snyk test --file=package-lock.json --dev`:

```
  ✗ Cross-site Scripting (XSS) [High Severity][https://snyk.io/vuln/SNYK-JS-SERIALIZEJAVASCRIPT-536840] in serialize-javascript@1.9.1
    introduced by uglifyjs-webpack-plugin@1.3.0 > serialize-javascript@1.9.1
  This issue was fixed in versions: 2.1.1
```
